src/lua/src/main.c

```diff
@@ -0,0 +1,186 @@
+#include <signal.h>
```

Comment:
please add license banner to the top and format the first part of the file to match the 2nd part. Like spaces after funcname and before (. Just to be consistent.